Index.html

```diff
@@ -1,14 +1,14 @@
 <!DOCTYPE html>
-<script src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
 <html>
-    <head><script>
-        $(function(){
-            $("#includeHtml").load("Navbar.html");
-        });
-    </script></head>
-  <link rel="stylesheet" href="Style.css" />
-  <div id="includeHtml"></div>
-  <p class="Hello">Hello!</p>
-
-  <h1 class="Heading">I am Anupama Jha</h1>
+  <head>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="Style.css" />
+  </head>
+  <body>
+    <header></header>
+    <main>
+    <p class="Hello">Hello!</p>
```

Comment:
use correct naming conventions for css class for example: "intro" "about" "flex" etc.